crates/bevy_xr_utils/src/lib.rs

```diff
@@ -1,6 +1,7 @@
 // use bevy::prelude::*;
 pub mod hand_gizmos;
-#[cfg(not(target_family = "wasm"))]
-pub mod xr_utils_actions;
+pub mod tracking_utils;
```

Comment:
this depends directly on openxr, it needs to be behind the wasm check
```suggestion
#[cfg(not(target_family = "wasm"))]
pub mod tracking_utils;
```